DetectorDescription/DDCMS/interface/DDDetector.h

```diff
@@ -34,6 +36,8 @@ namespace cms  {
     DDSpecParRegistry const& specpars() const {
       return m_specpars;
     }
+    
+    Volume worldVolume() const;
```

Comment:
Do you really want to return a copy of a `Volume` rather than a reference? `worldVolume` sounds large.

Reply:
Volume is a Handle to a pointer.